docs/snippets/common/storybook-coverage-addon-registration.ts.mdx

```diff
@@ -0,0 +1,16 @@
+```ts
+// .storybook/main.ts
+
+// Imports Storybook's configuration API
+import type { StorybookConfig } from '@storybook/types';
```

Comment:
Is this the correct way to import config types @kasperpeulen ? I thought it'd be coming from their framework e.g.

```ts
import type { StorybookConfig } from '@storybook/react-webpack5';
````

Reply:
@yannbf both work, but better be safe than sorry. I'll update the snippet to feature a more agnostic approach.

Reply:
Fixed with the latest commit